internal/pkg/aws/ecs/ecs.go

```diff
@@ -17,7 +18,12 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/exec"
 )
 
-const clusterStatusActive = "ACTIVE"
+const (
+	clusterStatusActive              = "ACTIVE"
+	waitServiceStablePollingInterval = 15 * time.Second
+	waitServiceStableMaxTry          = 40
```

Comment:
I honestly wouldn't be surprised if a deployment takes longer than 10 mins 😂 can we double this to 80?

Reply:
This is the same as what AWS CLI is doing right now. But sure we can adjust.